rdf-abac-core/src/test/java/io/telicent/jena/abac/bulk/BulkDirectory.java

```diff
@@ -63,25 +69,28 @@ public abstract class BulkDirectory {
 
     private static String level = null;
 
-    @BeforeAll public static void beforeClass() {
+    @BeforeAll
+    public static void beforeClass() {
         level = LogCtl.getLevel(BulkDirectory.LOG);
         LogCtl.setLevel(BulkDirectory.LOG, "warn");
     }
 
-    @AfterEach public void after() {
-        if (labelsStore instanceof LabelsStoreRocksDB rocksDB)
-        {
+    @AfterEach
+    public void after() {
+        if (labelsStore instanceof LabelsStoreRocksDB rocksDB) {
```

Comment:
It's sad how much this minor stylistic change makes me happy 👍

Reply:
Agreed